cmd/register/register_util.go

```diff
@@ -261,29 +283,35 @@ func DownloadFileFromHTTP(ctx context.Context, ref storage.DataReference) (io.Re
 }
 
 /*
-Get file list from the args list.
-If the archive flag is on then download the archives to temp directory and extract it.
+Get serialize output file list from the args list.
+If the archive flag is on then download the archives to temp directory and extract it. In case of fast register it will also return the compress source code
```

Comment:
```suggestion
If the archive flag is on then download the archives to temp directory and extract it. In case of fast register it will also return the compressed source code.
```